Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -237,8 +237,7 @@ def pull
         # Squash a Linuxbrew build-bottle-pr commit.
         if Utils.popen_read("git", "diff", orig_revision, "HEAD") =~ /^\+# .*: Build a bottle for Linuxbrew$/
           ohai "Squashing build-bottle-pr commit"
-          require "dev-cmd/squash-bottle-pr"
-          Homebrew.squash_bottle_pr
+          safe_system HOMEBREW_BREW_FILE, "squash-bottle-pr"
```

Comment:
`CoreTap` has a `ensure_installed!` method that unfortunately `Tap` does not have.
I decided to auto-tap `Linuxbrew/developer` rather than emit an error message.
See https://github.com/Linuxbrew/brew/pull/213